src/main/java/org/broadinstitute/hellbender/utils/activityprofile/BandPassActivityProfile.java

```diff
@@ -144,14 +144,14 @@ public double getSigma() {
         for ( final ActivityProfileState superState : super.processState(justAddedState) ) {
             if ( superState.isActiveProb() > 0.0 ) {
                 for( int i = -filterSize; i <= filterSize; i++ ) {
-                    final SimpleInterval loc = getLocForOffset(justAddedState.getLoc(), i);
```

Comment:
We should probably expand the test under `ActivityProfileUnitTest.testSoftClips()` to be extensive enough to catch this edge case. Maybe we should hard code some expected probability distributions or add a line asserting that the underlying simple locations for each state is not overlapping.